Mathlib/Analysis/InnerProductSpace/Basic.lean

```diff
@@ -1946,8 +2195,19 @@ theorem RCLike.inner_apply (x y : 𝕜) : ⟪x, y⟫ = conj x * y :=
   rfl
 #align is_R_or_C.inner_apply RCLike.inner_apply
 
-/-! ### Inner product space structure on subspaces -/
+end RCLike
+
+section PreSubmodule
```

Comment:
This `Pre` prefix in section names is just confusing. I know what you mean because of the context of this PR, but anyone else looking at this later is just going to ask: "what's a `PreSubmodule`?"

Reply:
I see, I just thought that the same section title for different assumptions isn't good. I change to `XXX_Seminormed`, is this better?